### ISSUE TITLE
Welcome to HSOC'22 - Important Points!

### ISSUE THREAD

**Issue by aaryahjolia** [aaryahjolia/dsa_competitive-coding#5]
Hey Everyone, Welcome to HSOC'22.

The Point System is decided as follows : 
HSOC22-Level-1   -  5 Points 
HSOC22-Level-2   -  10 Points
HSOC22-Level-3   -  20 Points
From My side, Contributing 2 code file will fetch you Level-1, 3 code files for Level-2 and atleast 5 code files for Level-3. 
(We can change Levels if you get it difficult to contribute this much!)

- Do not start your work until assigned. 
- If any issue is not assigned in 9 hours, Tag me on discord. (Except Night time :p )
- If your PR is not checked within 12 hours, Tag me on discord!
- Feel free to reach me on Discord for any kind of difficulties.
- Hope you have a great time in open source!

**Comment by Roshani9731** [comment]
Can I work on this ? I have to made readme file right?

**Comment by aaryahjolia** [comment]
> Can I work on this ? I have to made readme file right?

Readme file already exists. You can open a new issue to modify readme like adding more details or adding links to each folder, etc.
